internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -105,10 +113,25 @@ func (s *LoadBalancedWebService) Template() (string, error) {
 
 // Parameters returns the list of CloudFormation parameters used by the template.
 func (s *LoadBalancedWebService) Parameters() []*cloudformation.Parameter {
+	containerName := s.name
```

Comment:
What do you think about separating this out to its own method:
```golang
func (s *LoadBalancedWebService) TargetContainer() (*string, *int) {
//...
}
```

Reply:
done!